### PR TITLE
 Misc: Remove unnecessary assets and imports

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
issue #57 

closses #58

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+// import menu from "./menu.svg";
+// import close from "./close.svg";
+// import google from "./google.svg";
+// import apple from "./apple.svg";
+// import arrowUp from "./arrow-up.svg";
+// import discount from "./Discount.svg";
+// import facebook from "./facebook.svg";
+// import instagram from "./instagram.svg";
+// import linkedin from "./linkedin.svg";
+// import twitter from "./twitter.svg";
```
Comment (mittal-parth) [inline src/assets/index.js:17]: Why comments? Thats why the build is failing

Applicable everywhere

Comment (Brijeshthummar02) [reply src/assets/index.js:17]: i thought rather to remove lines i should comment it out which can be used in future if required, so do i have to remove it like whole line?

**Review by mittal-parth — CHANGES_REQUESTED** [review]
- Don't comment files that are required. That's breaking the build
- Commit messages are written in the imperative tense. For instance, Fix loading issue instead of fixed loading issue
- Prefix each commit with something that describes its category. Here it can be misc: <commit-message>
- When you mention 'closes #something' in the PR description, GitHub closes that issue automatically once merged. So specify the issue in this case not the PR number.

File: src/assets/index.js (hunk 1 above)
Comment (mittal-parth) [reply src/assets/index.js:17]: Remove the lines that are not required like binance, airbnb etc

But you have also commented out files that are required like insta, linkedIn etc

**Comment by Brijeshthummar02** [comment]
@mittal-parth req update.

**Comment by mittal-parth** [comment]
@Brijeshthummar02 As I said earlier also, you have to remove unnecessary imports also.

For instance, [here](https://github.com/mittal-parth/personal-portfolio/blob/main/src/assets/index.js#L4-L6)

```js
import robot from "./robot.png";
import send from "./Send.svg";
import shield from "./Shield.svg";

....

export {
  robot,
  send,
  shield,
  ...
}
```

These imports are not used beyond this file meaning they are never used. How did I find that? Go through each asset that is imported in this file. Do a global search for that (like robot here) in your code editor (Ctrl + Shift + F for VSCode on Linux/Windows). If that doesnt occur beyond this file, then its not being used. 

In contrast some file like `ethforall` is being imported and used in src/constants/index also which means it should not be removed.

**Comment by mittal-parth** [comment]
Hi @Brijeshthummar02, still working on it?

**Comment by Brijeshthummar02** [comment]
By mistake i deleted the repo from system so i can't edit the current pr need to create new if its okayy.

**Comment by mittal-parth** [comment]
@Brijeshthummar02 you don’t have to make a new PR. Your fork still exists on GitHub. Just pull that and you are good to go

**Comment by Brijeshthummar02** [comment]
@mittal-parth now is it ok?

**Comment by mittal-parth** [comment]
Have you checked if these are all imports to be removed? I just gave an example. 

Please read the entire [comment](https://github.com/mittal-parth/personal-portfolio/pull/59#issuecomment-2387631194) again and lemme know.

**Comment by mittal-parth** [comment]
Hi @Brijeshthummar02! Let’s close this soon? 
thanks!

**Comment by Brijeshthummar02** [comment]
i'm working on it.

**Comment by Brijeshthummar02** [comment]
give me complete list of which one to remove

**Comment by mittal-parth** [comment]
There is nothing left for you to do if I do that myself. The point of the issue was to explore and get a hang of the project. I have given enough steps for you to be able to do it comfortably.

If you won’t be able to pick it up, lemme know.

**Pull request closed without merging**